packages/profile-sync-controller/src/shared/encryption/cache.ts

```diff
@@ -6,8 +6,8 @@ type CachedEntry = {
   key: Uint8Array;
 };
 
-const MAX_PASSWORD_CACHES = 3;
-const MAX_SALT_CACHES = 10;
+const MAX_PASSWORD_CACHES = 100;
+const MAX_SALT_CACHES = 100;
```

Comment:
As discussed, I think this is valid.

If we find that this is a lot, we can decrease later.